.circleci/config.yml

```diff
@@ -14,6 +14,9 @@ executors:
       - image: tendermintdev/docker-website-deployment
     environment:
       AWS_REGION: us-east-1
+  protoc:
+    docker:
+      - image: tendermintdev/docker-protoc
```

Comment:
Is there a more generic docker image to use here? Also adding a hash or version here would be helpful.

Reply:
not sure what you mean by more generic. this is the simplest way to get buf into ci instead of installing it wach time

Reply:
I think @jackzampolin was asking if there's an image supported by buf team.

Reply:
testing one

Reply:
the issue is that the image is barebones and it seems buf requires go if you are using go extensions. reverted the change for now, but will look into it